pkg/client/unversioned/request.go

```diff
@@ -642,6 +642,7 @@ func (r *Request) Watch() (watch.Interface, error) {
 		return nil, err
 	}
 	if resp.StatusCode != http.StatusOK {
+		defer resp.Body.Close()
```

Comment:
yes. i think of it. i wonder whether it(`the resp`) will be  used out side of the transformResponse.